addons/account/views/tax_adjustments.xml

```diff
@@ -7,6 +7,5 @@
             <field name="view_mode">form</field>
             <field name="target">new</field>
         </record>
-        <!-- TODO: debug -->
-        <menuitem action="tax_adjustments_form" id="menu_action_tax_adjustment" parent="account.menu_finance_entries_actions" sequence="45" groups="account.group_account_manager" />
+        <menuitem action="tax_adjustments_form" id="menu_action_tax_adjustment" parent="account.menu_finance_entries_actions" sequence="45" groups="base.group_no_one" />
```

Comment:
I think `account.group_account_manager` shouldn't be removed from the groups; only those who have that right can see it in debug mode

Reply:
The parent menu "Accounting" already has the group "account.group_account_manager". No need to specify it explicitly.